.github/workflows/deploy.yaml

```diff
@@ -39,3 +40,39 @@ jobs:
       # Build and Deploy CDK application
       - name: Build & Deploy
         run: npm run cdk deploy tifeatures-timvt-staging -- --require-approval never
+
+  deploy_ecs_staging:
+    if: $NEW_ECS_DEPLOY == 'true'
```

Comment:
reviewer note: once we merge back to `main` we don't want to run any CD so this allows us to run it manually to test it out